Makefile

```diff
@@ -99,14 +99,8 @@ prtest: build check devicetests localtests
 # Installs the necessary Rust toolchains
 .PHONY: setup
 setup:
-	rustup toolchain add --profile minimal \
-		"$$(cat third_party/libtock-rs/rust-toolchain)"
-	rustup toolchain add --profile minimal \
-		"$$(cat third_party/tock/rust-toolchain)"
-	rustup target add --toolchain \
-		"$$(cat third_party/libtock-rs/rust-toolchain)" thumbv7m-none-eabi
-	rustup target add --toolchain \
-		"$$(cat third_party/tock/rust-toolchain)" thumbv7m-none-eabi
+	cd userspace && rustup target add thumbv7m-none-eabi && cd ..
```

Comment:
Better would be `(cd userspace && rustup target add thumbv7m-none-eabi)` -- that will execute the `cd` and the command in a sub-shell, so no matter what fails, you'll end up back in the directory where you started.  Same for below.  (Not sure if the parens are even necessary for Makefile commands, for which IIRC every command already gets executed in its own subshell, but not certain.)

Reply:
From what I gathered each "command" (i.e. the whole && chain) is executed in a sub shell. So it's actually redundant to do a "cd .." at the end. I'll remove it.